winforms/src/toga_winforms/widgets/dateinput.py

```diff
@@ -0,0 +1,55 @@
+import datetime
+
+from travertino.size import at_least
+
+from toga_winforms.libs import WinDateTime, WinForms
+
+from .base import Widget
+
+NO_MIN = WinForms.DateTimePicker.MinDateTime  # Year 1753
+NO_MAX = WinForms.DateTimePicker.MaxDateTime  # Year 9998
```

Comment:
Worth documenting these limits in the user-facing notes.

Reply:
See above.